Docs event history plans (#6700)
PipedreamHQ/pipedream: docs/docs/event-history/README.md

::: tip This feature is in beta

> Event Histories are currently in **beta** with a limited number of events to start.

Reviewer: ```suggestion
Event histories are currently in **beta** with a limited number of events to start.
```

Reviewer: Changed "Histories" to "histories" to follow Google's style guide on capitalization.

::: tip Beta access

> During the beta, all accounts will have access to view the past 7 days of events across all of their workflows. This limit will increase as this trial continues.

Reviewer: ```suggestion
During the beta, all accounts have access to view the past 7 days of events across all of their workflows. This limit increases as this trial continues.
```

Reviewer: Changed "will have" to "have" and "will increase" to "increases" to follow Google's style guide on future tense.

:::

The number of events recorded and available for viewing in the Event History depends on your plan. [Please see the pricing page](https://pipedream.com/pricing) for more details.

> ## Frequently Asked Questions

Reviewer: ```suggestion
## Frequently asked questions
```

Reviewer: Changed "Asked Questions" to "asked questions" to follow Google's style guide on capitalization.

> ### Are event histories available on all plans?

Reviewer: ```suggestion
### Are event histories available on all plans?
```

Reviewer: Changed "histories" to lowercase to follow Google's style guide on capitalization.

> Yes, event history is available for all workspace plans, including Free plans. However, the length of searchable or viewable history will change depending on your plan. [Please see the pricing page](https://pipedream.com/pricing) for more details.

Reviewer: ```suggestion
Yes, event history is available for all workspace plans, including free plans. However, the length of searchable or viewable history changes depending on your plan. [Please see the pricing page](https://pipedream.com/pricing) for more details.
```
